packages/gatsby/cache-dir/error-overlay-handler.js

```diff
@@ -1,43 +1,25 @@
-const overlayPackage = require(`@pmmmwh/react-refresh-webpack-plugin/overlay`)
-
-const ErrorOverlay = {
-  showCompileError: overlayPackage.showCompileError,
-  clearCompileError: overlayPackage.clearCompileError,
-}
-
 const errorMap = {}
 
-function flat(arr) {
-  return Array.prototype.flat ? arr.flat() : [].concat(...arr)
-}
-
 const handleErrorOverlay = () => {
   const errors = Object.values(errorMap)
-  let errorStringsToDisplay = []
+  let errorsToDisplay = []
   if (errors.length > 0) {
-    errorStringsToDisplay = flat(errors)
-      .map(error => {
-        if (typeof error === `string`) {
-          return error
-        } else if (typeof error === `object`) {
-          const errorStrBuilder = [error.text]
-
-          if (error.filePath) {
-            errorStrBuilder.push(`File: ${error.filePath}`)
-          }
-
-          return errorStrBuilder.join(`\n\n`)
-        }
-
-        return null
-      })
-      .filter(Boolean)
+    errorsToDisplay = errors.flatMap(e => e).filter(Boolean)
```

Comment:
Not necessary to do this handling here, I grab all info in the graphql-error component itself.
Also, we can use `flatMap` because we're now at Node 12.13 👍 and that's a Node 11 feature.